README.md

```diff
@@ -1 +1,166 @@
-# portfolio-desktop-version
\ No newline at end of file
+# Jose RM - Projects Portfolio
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] As requested by the previous reviewer, please create an header for your live demo and put a link to your live-demo underneath, The below is an example for you. 
- The live demo header from the template was originally under the key features:

<img width="911" alt="image" src="https://user-images.githubusercontent.com/52934128/215870676-6df1fca7-d891-46cb-879b-705f8b8f6fd5.png">



```
<!-- LIVE DEMO -->

## 🚀 Live Demo <a name="live-demo"></a>

> Add a link to your deployed project.

- [Live Demo Link](https://joramot.github.io/portfolio-desktop-version/)
```

Reply:
Ok, thank you @topeogunleye I removed  this section in the README template, that was a mistake.

Reply:
@joramot you're welcome